adyenofficial.php

```diff
@@ -816,6 +838,70 @@ public function displayForm()
             )
         );
 
+        $fields_form[1]['form'] = array(
+            'legend' => array(
+                'title' => $this->l('Payment method configurations'),
+                'image' => '../img/admin/edit.gif'
+            ),
+            'input' => array(),
+            'submit' => array(
+                'title' => $this->l('Save'),
+                'class' => 'btn btn-default pull-right'
+            )
+        );
+
+        // Apple pay merchant name input
+        $fields_form[1]['form']['input'][] = array(
+            'type' => 'text',
+            'label' => $this->l('Apple pay merchant name'),
```

Comment:
Just some caps on the labels
```suggestion
            'label' => $this->l('Apple Pay merchant name'),
```